pkgs/development/tools/git-ps-rs/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, libgpg-error
+, gpgme
+, dbus
+, openssl }:
```

```suggestion
{ lib
, fetchFromGitHub
, rustPlatform
, stdenv
, pkg-config
, libgpg-error
, gpgme
, dbus
, openssl
, Security
}:
```

Added `stdenv` and `Security`